nndownload/nndownload.py

```diff
@@ -148,6 +148,7 @@
             "quality": "super_high",
```

Comment:
This needs to be replaced by the highest quality that is acquired from params (`params['program']['stream']['maxQuality']`)

Reply:
Probably only the "official" ones can do. I can't find a 12M rn but here is a 8M one: https://live.nicovideo.jp/watch/lv347308094 Do note that the actual bitrate is often much lower.